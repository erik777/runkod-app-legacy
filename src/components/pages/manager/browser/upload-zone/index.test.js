import React from 'react';
import TestRenderer from 'react-test-renderer';

import UploadZone from './index';

test('1- render', () => {
  const props = {
    onDrop: () => {
    }
  };

  const renderer = TestRenderer.create(
    <UploadZone {...props}/>
  );

  expect(renderer.toJSON()).toMatchSnapshot();
});