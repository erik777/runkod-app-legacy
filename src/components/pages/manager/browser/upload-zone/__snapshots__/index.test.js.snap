// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- render 1`] = `
<div
  id="upload-zone"
>
  <div
    id="upload-zone-layer"
  />
  <div
    id="upload-zone-content"
  >
    <svg
      viewBox="0 0 512 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M416 199.5h-91.4V64H187.4v135.5H96l160 158.1 160-158.1zM96 402.8V448h320v-45.2H96z"
      />
    </svg>
     
    Drag and drop files or folders here to upload
  </div>
</div>
`;
